### ISSUE TITLE
Feature request: specify which of your accounts you want Blue Blocker to run on

### ISSUE THREAD

**Issue by scottdotjs** [kheina-com/Blue-Blocker#203]
Thank you so much for this incredible extension!

It would be really great if you were to add a setting to specify which accounts one wants Blue Blocker to run against, or the other way around, which accounts not to run against. I've found that on my general personal account, I want to block every blue user, but on my account for development topics, there are quite a few respectable people who've paid for the blue check for some reason. So ideally I could switch between logged-in accounts and Blue Blocker would check who I am when moving to the next queued user, and pause if it's an account I've requested no blocking on. Then when switched back to the other account, could reactivate.

**Comment by cooljeanius** [comment]
Yeah some statistics separation would be helpful, too, and if there are users in the queue, it would be good to know which account's blocklist they're in the queue for...

**Comment by rougetimelord** [comment]
The extension could check the value of `twid` in the cookie, then load/save configs/BlockQueues/BlockCounters in `browser.storage.local` with that value as the key. Not super sure how to implement that but it should be doable?

**Comment by cooljeanius** [comment]
Has duplicate #242 by @watatomo

**Comment by rougetimelord** [comment]
> Has duplicate #242 by @watatomo

I don't think that is a duplicate actually! This issue is about being able to use the extension on multiple accounts independently, but #242 would require syncing data across two (or more accounts)

**Comment by cooljeanius** [comment]
> > Has duplicate #242 by @watatomo
> 
> I don't think that is a duplicate actually! This issue is about being able to use the extension on multiple accounts independently, but #242 would require syncing data across two (or more accounts)

well, the reporter seemed to think it was the same thing, at least... eh, whatever...
